sktime/transformations/base.py

```diff
@@ -34,14 +34,25 @@
 
 
 class BaseTransformer(BaseEstimator):
-    """Transformer base class"""
+    """Transformer base class."""
+
+    # default tag values - these typically make the "safest" assumption
+    _tags = {
+        "univariate-only": False,  # can the transformer handle multivariate X?
+        "handles-missing-data": False,  # can estimator handle missing data?
+        "X-y-must-have-same-index": False,  # can estimator handle different X/y index?
+        "enforce_index_type": None,  # index type that needs to be enforced in X/y
```

Comment:
Just a detail, I think `enforce-index-type` would fit better with other tag names instead underscore?

Reply:
well, we're changing generally to underscore so we can use tags as kwarg names - that's not possible if a `-` appears in it. There were two inconsistent spellings so I went with the "target" spelling.